budgetApp1/budgetApp1/LoginViewController.swift

```diff
@@ -8,20 +8,19 @@
 import UIKit
 
 class LoginViewController: UIViewController {
-    @IBOutlet weak var usernameField: UITextField!
-    @IBOutlet weak var passwordField: UITextField!
+    @IBOutlet private weak var usernameField: UITextField!
+    @IBOutlet private weak var passwordField: UITextField!
     
-    var user1 = user.init(username: "", password: "", fullname: "", emailAddress: "")
+    var userTwo = User(username: "", password: "", fullname: "", emailAddress: "")
```

Comment:
good jib renaming the 2 to Two, but why is there even a user Two? why not just user?

Reply:
and this type of logic of instattiating a User should be done in the viewModel for this screen

Reply:
I think I'll change it later with storing user data using coredata.